src/elements/content-sidebar/AddTaskButton.js

```diff
@@ -29,7 +30,10 @@ class AddTaskButton extends React.Component<Props, State> {
         isDisabled: false,
     };
 
-    handleClickMenuItem = (taskType: TaskType) => this.setState({ isTaskFormOpen: true, taskType });
+    handleClickMenuItem = (taskType: TaskType) => {
+        this.props.onTaskModalOpen();
+        this.setState({ isTaskFormOpen: true, taskType });
```

Comment:
This change means we're tracking the modal's state in two separate places, in this component's state and in local storage. Can we unify them to avoid this duplication?

Reply:
Disregard, this comment was due to a misunderstanding on my part. Sorry about the confusion!